02-Reproject-Raster-In-R.Rmd

```diff
@@ -83,18 +91,23 @@ This lesson is a part of a series of raster data in R lessons:
 * [Lesson 06 - Plot Raster Time Series Data in R Using RasterVis and LevelPlot]({{ site.baseurl}}/R/Plot-Raster-Times-Series-Data-In-R/)
```

Comment:
Did i notice that on the previous lesson you removed the lesson numbers? whatever we do, it should be consistent.